frontend/src/app/shared/components/grids/widgets/news/news.component.html

```diff
@@ -18,7 +18,7 @@
         <op-principal
           *ngIf="news.author"
           [principal]="news.author"
-          [hide-name]="true"
+          [hideName]="true"
```

Comment:
I like to stay close to the way base html does it; lower case with dashes. Does this not work otherwise? Do we want to lint this as well?

Reply:
I changed this as `hide-name` conflicts with the angular style guide of avoiding input names: https://angular.io/guide/styleguide#avoid-aliasing-inputs-and-outputs

So generally, when following this rule, inputs always need to be camelCase without dashes, because they cannot be class field names otherwise